src/main/scala/viper/silver/testing/SilSuite.scala

```diff
@@ -49,7 +53,7 @@ abstract class SilSuite extends AnnotationBasedTestSuite with BeforeAndAfterAllC
     */
   override def beforeAll(configMap: ConfigMap) {
     this.configMap = configMap
-    verifiers foreach (_.start())
+    (verifiers ++ verifiers(configMap)) foreach (_.start())
```

Comment:
This looks fishy to me: `verifiers(configMap)` appears to return `verifiers`, in which case each verifier would be started twice.